integ/scripts/bash/run-e2e-automated.sh

```diff
@@ -30,19 +32,30 @@ refreshcreds() {
     unset CREDS
 }
 
+ASSUME_ROLE="false"
```

Comment:
The convention in bash is to only use a capitalized variable name for environment variables. If it isn't being exported, it should be lowercase.

Also, it you're using the string `"false"` instead of the boolean `false`, which I guess is fine, but not really what the intended purpose is.

Reply:
I've removed this variable since I realized it wasn't necessary after reading your comment. For some reason, I thought the `unset` would affect the parent environment even though it doesn't.